sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -69,7 +160,7 @@ export class AssessmentController {
    */
   private transformToAssessmentDTO(
     assessment: StudentAssessment,
-  ): ApplicationAssessmentWorkerOutDTO {
```

Comment:
Minor. What influenced to change WorkerIn/OutDTO to JobIn/OutDTO 😊

Reply:
The naming convention was actually changed during the previous PR but I missed a few files 😉 
Job and workers are both correct, but while writing more workers, "job" felt more logical and it is also shorter 😄 
For instance, a worker will process a job and the job has input/output parameters. Open to suggestions!
![image](https://user-images.githubusercontent.com/61259237/196827564-1e3b9077-4828-48e4-8e1a-ae850617aaf5.png)